### PR TITLE
Huddle landing page

### PR DESCRIPTION
**Pull request by lmizunoNorthern**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,9 +2,204 @@
...
+                         src="images/logo-black-and-white.svg"/>
+                </div>
+                <div class="col-12">
+                    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Animi consectetur dolore ipsa neque
+                        perferendis.</p>
+                </div>
+            </div>
+            <div class="row py-xl-5">
+                <div class="col-12">
+                    <p><img alt="" src="images/icon-phone.svg"/> Phone: +1-543-123-4567</p>
```
Comment (McJon) [inline index.html]: Add `tel` link

`<a href="tel:1-543-123-4567">1-543-123-4567</a>`

File: index.html [hunk 2]
```diff
@@ -2,9 +2,204 @@
...
+                </div>
+                <div class="col-12">
+                    <p><img alt="" src="images/icon-email.svg"/> example@complany.com</p>
```
Comment (McJon) [inline index.html]: A mailto link

<a href="mailto:1-543-123-4567">1-543-123-4567</a>

File: index.html [hunk 3]
```diff
@@ -2,9 +2,204 @@
...
+                </div>
+                <div class="col-10 col-md-6">
+
+                    <i class="fab fa-facebook-square social-media"></i>
```
Comment (McJon) [inline index.html]: These should be linked to some socials can just be facebook.com

Make sure you set the target to it opens a new link.

File: index.html [hunk 4]
```diff
@@ -2,9 +2,204 @@
...
+                    </div>
+                </div>
+
+
+            </div>
+        </div>
+        <div class="col-12 col-sm-10 mt-5 mt-md-0 col-md-5 order-2 order-md-1">
+            <div class="row py-xl-3">
+                <div class="col-8 col-sm-5 ">
+                    <img class="w-100" alt="Website logo, huddle."
```
Comment (McJon) [inline index.html]: Logos should always link back to the homepage.

File: index.html [hunk 5]
```diff
@@ -2,9 +2,204 @@
...
+                    <p>To recieve tips on how to grow your community, sign up to our weekly newsletter. We’ll never send
+                        you spam or pass on your email address</p>
+                </div>
+                <div class="col-12">
+                    <div class="row d-md-flex align-content-md-stretch">
+                        <div class="col-12 col-md-7 col-lg-8">
+                            <input type="email" class="input-footer">
+                        </div>
+                        <div class="col-6 col-md-5 col-lg-4 ml-lg-auto mt-2 mt-md-0">
+                            <button class="newsl_button">Subscribe</button>
```
Comment (McJon) [inline index.html]: type="submit"

File: index.html [hunk 6]
```diff
@@ -2,9 +2,204 @@
...
+    <link rel="stylesheet" href="css/main-style.css">
+    <link rel="stylesheet" href="css/bootstrap-grid.css">
+    <link rel="stylesheet" href="css/bootstrap-reboot.css">
+    <link rel="stylesheet" href="css/bootstrap.css">
+    <link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css"
+          integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
+    <link href="https://unpkg.com/ionicons@4.4.4/dist/css/ionicons.min.css" rel="stylesheet">
+
```
Comment (McJon) [inline index.html]: Look into adding a favicon

File: index.html [hunk 7]
```diff
@@ -2,9 +2,204 @@
...
+
```
Comment (McJon) [inline index.html]: Look into adding chrome's theme color https://developers.google.com/web/updates/2014/11/Support-for-theme-color-in-Chrome-39-for-Android

**Comment by McJon** [comment]
Try to pass achecker https://achecker.ca/checker/index.php
as well as chrome's accessibility audit and any other tools you can find to help.

**Comment by lmizunoNorthern** [comment]
> Try to pass achecker https://achecker.ca/checker/index.php
> as well as chrome's accessibility audit and any other tools you can find to help.

Fixed, acheker scores are good, CSS and HTML validation are intact and it is now AA compliant.

File: index.html (hunk 1 above)
Comment (lmizunoNorthern) [reply index.html]: fixed

File: index.html (hunk 2 above)
Comment (lmizunoNorthern) [reply index.html]: fixed

File: index.html (hunk 3 above)
Comment (lmizunoNorthern) [reply index.html]: fixed

File: index.html (hunk 4 above)
Comment (lmizunoNorthern) [reply index.html]: fixed

File: index.html (hunk 6 above)
Comment (lmizunoNorthern) [reply index.html]: fixed

File: index.html (hunk 7 above)
Comment (lmizunoNorthern) [reply index.html]: fixed

File: index.html (hunk 5 above)
Comment (lmizunoNorthern) [reply index.html]: fixed

File: index.html (hunk 4 above)
Comment (McJon) [reply index.html]: Footer tooo, and add a bottom margin to the logo.

**Comment by McJon** [comment]
Add a form tag around the newsletter form so the submit will "Work"

File: index.html [hunk 8]
```diff
@@ -181,17 +187,25 @@ <h5>
...
+                        <p><img alt="" src="images/icon-phone.svg"/> Phone: +1-543-123-4567</p>
+                    </a>
                 </div>
                 <div class="col-12">
-                    <p><img alt="" src="images/icon-email.svg"/> example@complany.com</p>
+                    <a href="mailto:example@complany.com">
+                        <p><img alt="" src="images/icon-email.svg"/> example@complany.com</p>
+                    </a>
                 </div>
                 <div class="col-10 col-md-6">
```
Comment (McJon) [inline index.html]: - col-md-7
- remove the height on these `fab`'s
- negative left margin to line up with the lines above.

File: index.html [hunk 9]
```diff
@@ -148,19 +153,20 @@ <h1 class="text-center">
...
+                    </h3>
                     <p>To recieve tips on how to grow your community, sign up to our weekly newsletter. We’ll never send
                         you spam or pass on your email address</p>
                 </div>
                 <div class="col-12">
                     <div class="row d-md-flex align-content-md-stretch">
                         <div class="col-12 col-md-7 col-lg-8">
-                            <input type="email" class="input-footer">
+                            <label class="input-footer" for="email-news" title="Email">
+                                <input id="email-news" type="email" class="input-footer"></label>
```
Comment (McJon) [inline index.html]: Add some horizontal padding.

**Comment by McJon** [comment]
If you're up for it, look into deploying this to firebase!

File: index.html (hunk 9 above)
Comment (lmizunoNorthern) [reply index.html]: Added

File: index.html (hunk 8 above)
Comment (lmizunoNorthern) [reply index.html]: Fixed

**Comment by lmizunoNorthern** [comment]
> Add a form tag around the newsletter form so the submit will "Work"

Fixed

File: index.html (hunk 4 above)
Comment (lmizunoNorthern) [reply index.html]: Fixed

**Comment by McJon** [comment]
Reviewed!

**Pull request merged**